sonic-net/sonic-buildimage
# Upgrade Ansible to 6.7.0 and make Python3 as the default interpreter in sonic-mgmt-docker

#### Why I did it
This PR is part of sonic-mgmt-docker Python3 migration project.

##### Work item tracking
- Microsoft ADO **(number only)**: 24397943

#### How I did it
1. Upgrade Ansible to 6.7.0
2. Make Python3 as the default interpreter. python is a soft link to python3. If you want to use python2, use the command python2 explicitly.
3. Upgrade some pip packages to higher version in order to meet security requirement.

#### How to verify it
1. Build a private sonic-mgmt-docker successfully.
2. Verify python is python3.
4. Verify python2 is working with 202012 and 202205 branch.
5. Verify python3 is working with master branch.



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
